common/src/main/java/com/wynntils/features/map/MainMapFeature.java

```diff
@@ -94,7 +88,7 @@ public class MainMapFeature extends Feature {
     public final Config<Boolean> autoWaypointChests = new Config<>(true);
 
     @Persisted
-    public final Config<LootChestType> minTierForAutoWaypoint = new Config<>(LootChestType.TIER_3);
+    public final Config<LootChestTier> minTierForAutoWaypoint = new Config<>(LootChestTier.TIER_3);
```

Comment:
I think this should go away. Once we started separating collected chest locations from user added waypoints, we should collect all of them. Then we can chose which to *display* in the map, but that is a different story.

Reply:
That way you can e.g. show tier-1 chests in the minimap, but only 3-4 on the main map. And if we start collecting tier1 right away, we have a bunch of chests to display if/when the user changes her mind and wants to display chests of a lower tier. So the collection should not be tied to what we display. (Which it is right now)

Reply:
Agreed, I can remove this.